tests/common/ixia/ixia_fixtures.py

```diff
@@ -224,17 +249,13 @@ def ixia_testbed(conn_graph_facts,
 
     ixia_ports = ixia_fanout_list.get_ports(peer_device=duthost.hostname)
 
-    ports = list()
-    port_names = list()
     port_speed = None
 
     """ L1 config """
+    config = snappi_api.config()
```

Comment:
let's leave this method intact and have the snappi setup done in a separate method